src/aa_py_filterbank_metadata.cpp

```diff
@@ -21,6 +22,7 @@ namespace astroaccelerate {
 	obj->m_nbits = nbits;
 	obj->m_nsamples = nsamples;
 	obj->m_nchans = nchans;
+	obj->m_src_raj = src_raj;
```

Comment:
Same as before. I do not know what it should represent.

Reply:
As before mentioned:
m_src_raj = J2000 Right Ascension (hh:mm:ss.ssss)
later if we extend also
m_src_dej = J2000 Declination     (dd:mm:ss.ssss)